build/docker/files/root/docker-entrypoint-initdb.d/marti-init.sql

```diff
@@ -3,16 +3,6 @@ CREATE TABLE `dice_emails` (
   UNIQUE KEY `registered_email` (`registered_email`)
 ) DEFAULT CHARSET=latin1;
 
-CREATE TABLE `dice_table` (
-  `request_ID` int(11) NOT NULL AUTO_INCREMENT,
-  `UUID` varchar(36) NOT NULL,
-  `dice` varchar(256) NOT NULL,
-  `subject` varchar(70) NOT NULL,
-  `time_stamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
-  PRIMARY KEY (`request_ID`),
-  KEY `UUID` (`UUID`)
-) AUTO_INCREMENT=57 DEFAULT CHARSET=latin1;
-
```

Comment:
Yeah, noticed that as well...
There is an empty database on the warclub server ^^